dev/annotate_source.py

```diff
@@ -220,8 +220,6 @@ def tag_head(self):
       # We didn't manually specify, but want to force a rebuild of the old tag.
       elif self.__force_rebuild:
```

Comment:
This only happens when head and the last tag are equal?

Reply:
Yes, otherwise it's a new semver because there are changes since the last tag.